iryna_kurhuzenkava/14.ansible.ws/ans_ws/roles/redmine_db/tasks/main.yaml

```diff
@@ -0,0 +1,20 @@
+---
+- mysql_db:
+    name: "{{ app_db_name }}"
+    encoding: utf8
+    login_unix_socket: /var/run/mysqld/mysqld.sock
+
+- mysql_user:
+    name: "{{ app_db_user }}"
+    password: "{{ app_db_pass }}"
+    priv: "{{ app_db_name }}.*:ALL"
+    login_unix_socket: /var/run/mysqld/mysqld.sock
+  no_log: yes
+
+- name: Config database
```

Comment:
Please move this task to app role, and redeploy it to check if everything is correct